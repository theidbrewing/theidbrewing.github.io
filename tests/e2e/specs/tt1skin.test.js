import { visitAdminPage, createNewPost, clickBlockAppender, getEditedPostContent, activateTheme } from '@wordpress/e2e-test-utils';

describe('TT1 Skin', () => {

    // it('Sample Test', async () => {
    //     await visitAdminPage( '/' );
    //     expect(true).toEqual(true);
    // });

    beforeEach(async () => {
        await activateTheme('twentytwentyone');
        await createNewPost();
    });

    it('should have the heading block custom style', async () => {
        await clickBlockAppender();
        await page.keyboard.type('## 2');
        //select custom style
        await page.click(
            `.block-editor-block-styles__item[aria-label="section title"]`
        );
        expect(await getEditedPostContent()).toMatchSnapshot();
    });

});
