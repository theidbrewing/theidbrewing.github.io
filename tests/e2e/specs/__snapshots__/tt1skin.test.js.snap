// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TT1 Skin should have the heading block custom style 1`] = `
"<!-- wp:heading {\\"className\\":\\"is-style-section-title\\"} -->
<h2 class=\\"is-style-section-title\\" id=\\"2\\">2</h2>
<!-- /wp:heading -->"
`;
